### PR TITLE
show how to exit nix repl

### PR DESCRIPTION
**Pull request by otti0815**
I came across this while doing the tutorial

### CODE REVIEW

**Comment by otti0815** [comment]
I have read the CLA Document and I hereby sign the CLA

File: source/tutorials/nix-language.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ nix-repl> { a.b.c = 1; }
 nix-repl> :p { a.b.c = 1; }
 { a = { b = { c = 1; }; }; }
 ```
+
+to exit [`nix repl`][nix-repl] use `:q`
```
Comment (fricklerhandwerk) [inline source/tutorials/nix-language.md:133]: Also this line should go *above* the note, not inside it. It has nothing to do with the note.

```suggestion

Type `:q` to exit [`nix repl`][nix-repl].

```

**Review by fricklerhandwerk — CHANGES_REQUESTED** [review]
@otti0815 thanks for the feedback!

I'm not sure if we really need this. The first line of output says to type `:?` for help, and that lists all commands, including `:q`. Admittedly this requires additional steps, and more careful reading, which strains the already strained cognitive capacity of people trying to learn new things.

Therefore it's probably good to have the information in context right there. I made a suggestion where I think it would be most helpful.

If you'd like to contribute more, here's an idea that could make the REPL experience more approachable for beginners: https://github.com/NixOS/nix/issues/7646

**Review by fricklerhandwerk — APPROVED** [review]

**Comment by fricklerhandwerk** [comment]
Thanks for the contribution. Hope to see you around again if you find more things that can be improved!

**Pull request merged**